policyfile/chef_work_environment.rb

```diff
@@ -0,0 +1,34 @@
+name 'chef_work_environment'
+
+default_source :supermarket, 'https://supermarket.chef.io'
+cookbook 'chef-client', '~> 12.3.3'
+cookbook 'chef_work_environment', '~> 0.2.0'
+
+run_list [
+  'chef-client',
+  'chef_work_environment',
+  'chef_work_environment::vault'
+]
+
+default['chef_client'] = {
+  'interval': 1800,
+  'splay': 30,
+}
+
+default['chef_work_environment'] = {
+  'chef_workstation': {
+    'version': '21.2.303'
+  }
+}
```

Comment:
This example is a bit confusing at first sight. Probably make it a bit nicer when adding some notes to it as well?
Something like `the policygroup is named chef_work_environment and sets attribute overrides for chef_workstation with version`.